web-registry/src/components/organisms/MyProjects.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { Box, Grid } from '@mui/material';
+// import { useNavigate } from 'react-router-dom'; TODO Issue: regen-network/regen-registry#913
+
+import CreateProjectCard from 'web-components/lib/components/cards/CreateProjectCard';
+import ErrorBanner from 'web-components/lib/components//banner/ErrorBanner';
+
+import { useWallet } from '../../../src/lib/wallet';
+import {
+  useCreateProjectMutation,
+  useGetWalletByAddressQuery,
+  useCreateWalletMutation,
+} from '../../generated/graphql';
+
+const MyProjects: React.FC = () => {
+  const [error, setError] = useState<string | null>(null);
+  const { wallet } = useWallet();
+  // const navigate = useNavigate(); TODO Issue: regen-network/regen-registry#913
+  const [createProject] = useCreateProjectMutation();
+  const [createWallet] = useCreateWalletMutation();
+  const { data: walletData } = useGetWalletByAddressQuery({
```

Comment:
This will change with Kyle's recent updates. Will update this thread when ready.